index.html

```diff
@@ -200,6 +221,94 @@ <h3>Profesional Art <br>
             <button class="see-project-btn" type="button" autofocus>See Project </button>
           </article>
         </div>
+
+        <!-- new created desktop work cards part -->
+
+        <div class="desktop-cards">
+          <article class="desktop-card-1">
+            <button type="button" autofocus>See Project </button>
+          </article>
+          <article class="desktop-card-2 desk-card">
```

Comment:
- [ ] As stated by the previous reviewer please 😃try to add more sides margins to these cards in order to match the Figma design I suggest for you wrap the content inside the container parent and give it the margin or padding to be like the Figma Design😊

> Check the table below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/214833168-086db073-43d2-4250-b091-5157fb66c56b.png)|   ![image](https://user-images.githubusercontent.com/83360636/214833265-cbe8b141-baac-4b5b-b924-e224cc2d661e.png)|